### PR TITLE
agent: Small code rework

### PR DESCRIPTION
**Pull request by otavio**
This PR is all about code rework for cleanness so it is not intended to add new features or do massive reworks, just small ones.

### CODE REVIEW

**Comment by otavio** [comment]
@sixhills please take a look here as well.

**Comment by sixhills** [comment]
Excellent! Comments help to understand the way the code works.

File: agent/main.go [hunk 1]
```diff
@@ -102,12 +122,22 @@ func main() {
 				continue
 			}
 
+			namespace := agent.authData.Namespace
+			tenantName := agent.authData.Name
+			sshEndpoint := agent.serverInfo.Endpoints.SSH
+
+			sshid := strings.NewReplacer(
```
Comment (gustavosbarreto) [inline agent/main.go:129]: Why not? 
```fmt.Sprintf("%s.%s@%s", namespace, tenantName, strings.Split(sshEndpoint, ":")[0])```

**Review by gustavosbarreto — APPROVED** [review]

File: agent/main.go (hunk 1 above)
Comment (otavio) [reply agent/main.go:129]: Even nicer. :-)

**Pull request merged**